pages/common/tex-fmt.md

```diff
@@ -0,0 +1,16 @@
+# tex-fmt
+
+> Format LaTeX source code.
+> More information: <https://github.com/WGUNDERWOOD/tex-fmt>.
+
+- Format a file, overwriting the original file:
+
+`tex-fmt {{path/to/file.tex}}`
+
+- Check if a file is correctly formatted:
+
+`tex-fmt --check {{path/to/file.tex}}`
+
+- Format a file read from `stdin` and print to `stout`:
```

Comment:
```suggestion
- Format a file read from `stdin` and print to `stdout`:
```

One more typo I previously overlooked... LGTM after that. Thanks for incorporating my previous suggestions!

Reply:
Thanks, fixed